tests/test_models.py

```diff
@@ -1,20 +1,20 @@
-#/usr/bin/env python3
+# /usr/bin/env python3
 # we do this to enable deterministic behavior on the gpu, see
 # https://docs.nvidia.com/cuda/cublas/index.html#cublasApi_reproducibility for
 # details
-import os
-os.environ['CUBLAS_WORKSPACE_CONFIG'] = ':4096:8'
-import matplotlib.pyplot as plt
-import plenoptic
-import plenoptic as po
-import pytest
-import numpy as np
+from conftest import DEVICE, DATA_DIR
+from test_metric import osf_download
+import os.path as op
 import scipy.io as sio
+import pyrtools as pt
+from plenoptic.simulate.canonical_computations import gaussian1d, circular_gaussian2d
+import plenoptic as po
 import torch
-import os.path as op
-from test_metric import osf_download
-from plenoptic.simulate.canonical_computations import (gaussian1d, circular_gaussian2d)
-from conftest import DEVICE, DATA_DIR
+import numpy as np
+import pytest
+import matplotlib.pyplot as plt
+import os
+os.environ['CUBLAS_WORKSPACE_CONFIG'] = ':4096:8'
```

Comment:
what is this for? I forget, could we add a comment?

Reply:
I don't know what this is. These changes are just my editor automatically rearranging the imports.

Reply:
gotcha. we should probably have a PR (probably as part of running python black?) that standardizes the imports.